airbyte-integrations/connectors/source-greenhouse/unit_tests/conftest.py

```diff
@@ -2,28 +2,3 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
```

Comment:
@darynaishchenko should we just remove the whole file then? It's just the header comment left.

Reply:
removed